packages/epo-react-lib/src/atomic/Video/patterns/Generic/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FunctionComponent } from "react";
+import { Trans } from "react-i18next";
+import * as Styled from "./styles";
+
+export interface GenericVideoProps {
```

Comment:
any other props we'd want to send to a generic video player? Or configurability? This is very geared towards Canto's setup of sending a scaled down mp4 with no additional sources or subtitle tracks.